cmd/contextList.go

```diff
@@ -0,0 +1,27 @@
+package cmd
+
+import (
+	"github.com/jedib0t/go-pretty/table"
+	"github.com/netsage-project/grafana-dashboard-manager/config"
+	"github.com/spf13/cobra"
+)
+
+var contextList = &cobra.Command{
+	Use:   "list",
+	Short: "List context",
+	Long:  `List contexts.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		tableObj.AppendHeader(table.Row{"context", "active"})
+		contexts := config.GetContexts()
+		activeContext := config.GetContext()
+		for _, item := range contexts {
```

Comment:
nit: for _, item := range config.GetContexts()

Another nit: What about having config.GetContextMap() that returns the map[string]Context. Then you can range directly over the keys and not have to use the funk helper.

Reply:
that feels like it's exposing too much internal data for no reason.  Plus funk I think provides some basic functionality that Go should just have IMO.

Reply:
👍